guides/v3.4.0/models/index.md

```diff
@@ -1,5 +1,19 @@
-Models are objects that represent the underlying data that your
-application presents to the user. Different apps will have very
+Ember developers have great options for how they handle data from back end APIs. 
+Ember itself works with any type of back end: REST, JSON API, GraphQL, or anything else.
+
+Many developers choose to use Ember Data, a powerful set of tools for formatting requests, serializing responses, and efficiently managing a local cache of data. 
+The Ember Data library is included by default in Ember apps. 
+Alternately, some developers write all their own code to handle API requests, using native JavaScript methods like [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) or third-party libraries.
+Many apps use a combination of approaches.
+
+This section of the Guides describes the essential features of Ember Data. To learn about other ways to handle data in an app, check out [Ember Observer](https://www.emberobserver.com/) for addons or search for community-made tutorials.
```

Comment:
I'm not sure suggesting a visit to ember observer should be the first recommendation to someone interested in alternatives to using `ember-data`.  We may not want to reference specific 3rd party tutorials, either though.  Maybe just remove the whole 'to learn about other ways...' sentence?

Reply:
I think it’s important to show that people have options. Sometimes people abandon ember because they think they don’t, and other times people think they need to write everything themselves. I want to present the realistic range, while still encouraging the use of Ember Data. I will reword some.